Ag 14135 touch gestures

kylerphillips opened the pull request:
![Screenshot 2025-04-28 at 12 11 43](https://github.com/user-attachments/assets/997f3df7-976d-4c3a-95a0-38b26cc105e4)

kylerphillips committed:
AG-14135 Update touch examples

kylerphillips committed:
AG-14135 Responsive and hr tweaks

kylerphillips committed:
AG-14135 Rename touchDevice to touchImage

kylerphillips committed:
AG-14135 Rename src refs

kylerphillips committed:
AG-14135 Fix linting

kylerphillips committed:
AG-14135 Remove unused css

markdurrant commented on packages/ag-charts-website/src/content/docs/touch/_examples/single-finger-touch-dragging/style.css:
```diff
@@ -0,0 +1,86 @@
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+}
```
Think we can remove this file now.

taktran reviewed (approved):
👍🏼

markdurrant commented on external/ag-website-shared/src/components/touch-image/TouchImage.module.scss:
```diff
@@ -0,0 +1,102 @@
+            display: none;
+        }
+    }
+}
```
A few things to look at in this file. 

- Do we need all this nesting? I think most classes should be ok on their own. 
- Generally prefer nesting `#{$selector-darkmode} &` & media queries 
- `.deviceContent` class not used?
- `button:not(ul button)` & `header` not used.

markdurrant reviewed (changes requested):
Looking good on site. But a few things to refactor for PR. 

Not sure if we want to keep those MP4s in the Repo if they're not being used on the website directly.

kylerphillips committed:
AG-14135 Clean up css nesting

kylerphillips commented:
> Looking good on site. But a few things to refactor for PR.
> 
> Not sure if we want to keep those MP4s in the Repo if they're not being used on the website directly.

- Are you referring to the gifs here?

- I've cleaned up the css nesting and removed unused lines. I couldn't seem to get the dark mode nesting to work inside a single `.touchImage` declaration, something strange happening there. The nesting won't work inside the media queries as those are for mobile, let me know if I misunderstood though.
